tests/integ/modin/frame/test_apply_axis_0.py

```diff
@@ -323,6 +323,7 @@ def test_groupby_apply_constant_output():
         snow_df,
         native_df,
         lambda df: df.groupby(by=["fg"], axis=0).apply(lambda x: [1, 2]),
+        test_attrs=False,  # native pandas is inconsistent about whether it propagates attrs
```

Comment:
What does inconsistent mean here?

Reply:
There are many similar occurrences in the other tests. Can we clarify the native pandas inconsistency there as well?

Reply:
By "inconsistent," I mean that some invocations of the function will propagate while some will not, depending on the arguments provided. In this particular test case, I think `attrs` isn't propagated because there's no relationship between the return of the applied lambda `lambda x: [1, 2]` and the data in the original frame; we don't really have a good way to check for this.

There are some cases we _can_ check:
```
>>> native_df = native_pd.DataFrame({"a": [1,2,3], "b": [4,5,6]})
>>> native_df.attrs = {"k": "v"}
>>> native_df.groupby("a").apply(sum).attrs  # function argument causes reset
{}
>>> native_df.groupby("a").apply("sum").attrs  # string argument causes propagation
{'k': 'v'}
```

Here, I think it's possible to tell based on the type of the provided aggregation, but others are not possible to determine or would require us to look at the input frame, and I don't think it's worth the effort to identify every single one of these cases in native pandas.

Do you have any thoughts on how to make this more clear?

Reply:
Thanks for the explanation. Yes, I don't think it's worth it. But in terms of clarifying the comment, maybe it's worth adding what you mentioned above: "Some invocations of the function will propagate while some will not, depending on the arguments provided."

Reply:
Sounds good, updated comments to be more clear.